api/users/pagination.py

```diff
@@ -0,0 +1,60 @@
+from collections import OrderedDict
+from django.core.urlresolvers import reverse
+
+from api.base.pagination import JSONAPIPagination
+from rest_framework.response import Response
+
+class UserNodeLogPagination(JSONAPIPagination):
+
+    node_log_aggregates = None
```

Comment:
Yeah, that is set to none so the attribute exists, and the view can attached the get_aggregates method to the pagination class.